pysmt/test/smtlib/omt/smtlib2_boxed_int.smt2

```diff
@@ -0,0 +1,66 @@
+; -*- SMT2 -*-
+;
+; Author: Patrick Trentin <patrick.trentin@unitn.it>
+;
+; This file is part of OptiMathSAT.
```

Comment:
Same for all the other files: say that we took these files from the optimathsat distribution with the permission of the authors.

Reply:
And maybe remove the rest of the intro comments